docs/ai/quickstarts/quickstart-local-ai.md

```diff
@@ -0,0 +1,168 @@
+ms.date: 06/04/2024
+ms.topic: quickstart
+ms.custom: devx-track-dotnet, devx-track-dotnet-ai
+author: alexwolfmsft
+ms.author: alexwolf
+---
+
+# Chat with a local AI model using .NET and Semantic Kernel
+
+Local AI models provide powerful and flexible options for building AI solutions. In this quickstart, you'll explore how to set up and connect to a local AI model using .NET and the Semantic Kernel SDK. For this example, you'll run the local AI model using Ollama.
+
+## Prerequisites
+
+* [Install .NET 8.0](https://dotnet.microsoft.com/download) or higher
+* [Install Ollama](https://ollama.com/) locally on your device
+
+## Run the local AI model
+
+Complete the following steps to configure and run a local AI Model on your device. Many different AI models are available to run locally and are trained for different tasks, such as generating code, analyzing images, generative chat, or creating embeddings. For this quickstart, you'll use the general purpose `phi3:mini` model, which is a small but capable generative AI created by Microsoft.
+
+1. Open a terminal window and verify that Ollama is available on your device:
+
+    ```bash
+    ollama
+    ```
+
+    If Ollama is running, it displays a list of available commands.
+
+1. Pull the `phi3:mini` model from the Ollama registry and wait for it to download:
+
+    ```bash
+    ollama pull phi3:mini
+    ```
+
+1. After the download completes, run the model:
+
+    ```bash
+    ollama run phi3:mini
+    ```
+
+    Ollama starts the `phi3:mini` model and provides a prompt for you to interact with it.
+
+## Create the .NET app
+
+Complete the following steps to create a .NET console app that will connect to your local `phi3:mini` AI model:
+
+1. In a terminal window, navigate to an empty directory on your device and create a new app with the `dotnet new` command:
+
+    ```dotnetcli
+    dotnet new console
+    ```
+
+1. Add the Semantic Kernel SDK package to your app:
+
+    ```dotnetcli
+    dotnet add package Microsoft.SemanticKernel
+    ```
+
+1. Open the new app in your editor of choice, such as Visual Studio Code.
+
+    ```dotnetcli
+    code .
+    ```
+
+## Connect to and chat with the AI model
+
+The Semantic Kernel SDK provides many services and features to connect to AI models and manage interactions. In the steps ahead, you'll create a simple app that connects to the local AI and stores conversation history to improve the chat experience.
+
+1. Open the _Program.cs_ file and replace the contents of the file with the following code:
+
+    ```csharp
+    using Microsoft.SemanticKernel;
+    using Microsoft.SemanticKernel.ChatCompletion;
+    
+    // Create a kernel with OpenAI chat completion
+    #pragma warning disable SKEXP0010
+    Kernel kernel = Kernel.CreateBuilder()
+                        .AddOpenAIChatCompletion(
+                            modelId: "phi3:mini",
+                            endpoint: new Uri("http://localhost:11434"),
```

Comment:
Why use this endpoint, it's not mentioned anywhere? Does `ollama` listening on this port or something?

Reply:
It's the default endpoint that Ollama starts up on after a standard install - I will add a note about this, good catch.